microcosm_pubsub/conventions/messages.py

```diff
@@ -14,10 +14,6 @@ class URIMessageSchema(PubSubMessageSchema):
     the consumer can retry a few times before giving up (e.g. via SQS dead-lettering).
 
     """
-    def __init__(self, media_type, **kwargs):
-        super().__init__(**kwargs)
-        self.MEDIA_TYPE = media_type
```

Comment:
There is an argument for making `media_type` an optional argument for backward compatibility. However, I believe this hasn't been needed since early 2017; I'd rather clean this up properly.